deployment/main.tf

```diff
@@ -0,0 +1,182 @@
+# TODO(michaelkedar): Organise / refactor logically into multiple tf files
+# and possibly add variables for some field values.
+
+# APIs
+# TODO(michaelkedar): Check whether any required apis are missing.
+
+resource "google_project_service" "compute_engine_api" {
+  service = "compute.googleapis.com"
+}
+
+resource "google_project_service" "kubernetes_engine_api" {
+  service = "container.googleapis.com"
+}
+
+# Network
+
+resource "google_compute_subnetwork" "my-subnet-0" {
+  name                     = "my-subnet-0"
+  network                  = "default"
+  ip_cidr_range            = "10.45.32.0/22"
+  private_ip_google_access = true
+  region                   = "us-central1"
+}
+
+resource "google_compute_router" "router" {
+  name    = "router"
+  network = "default"
+  region  = "us-central1"
+}
+
+resource "google_compute_router_nat" "nat-config" {
```

Comment:
very nice! did you test that internet access works?

Reply:
Yep! The worker was able to pull the oss-fuzz repo from github